documentation/modules/configuring/con-config-kafka-connector.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// assembly-config.adoc
+
+[id='con-kafka-connector-config-{context}']
+= Configuring Kafka Connect connectors
+
+[role="_abstract"]
+The `KafkaConnector` resource provides a Kubernetes-native approach to management of connectors by the Cluster Operator.
+To create, delete, or reconfigure connectors with `KafkaConnector` resources, you must set the `use-connector-resources` annotation to `true`  in your `KafkaConnect` custom resource.
+
+.Annotation to enable KafkaConnectors
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaConnectApiVersion}
+kind: KafkaConnect
+metadata:
+  name: my-connect-cluster
+  annotations:
+    strimzi.io/use-connector-resources: "true"
+  # ...
+----
+
+When the use-connector-resources annotation is enabled in your KafkaConnect configuration, you must define and manage connectors using `KafkaConnector` resources. These resources must be deployed in the same namespace as the Kafka Connect cluster they are associated with, and the Kafka cluster interacting with the external data system.
+
+NOTE: Alternatively, you can manage connectors using the Kafka Connect REST API instead of `KafkaConnector` resources.
+To use the API, you must remove the annotation to use `KafkaConnector` resources in the `KafkaConnect` the resource.
+
+`KafkaConnector` resources are configured to connect to external systems and are deployed within the same Kubernetes cluster as the Kafka Connect and Kafka clusters that interact with the external data systems.
```

Comment:
This does not seem much sense. The connectors are not deployed in the same Kubernetes cluster as the Kafka Connect. They are deployed inside the Kafka Connect. The Kafka cluster can actually run somewhere completely else.

Reply:
Yeah. It's a bit confused.  I've changed as follows:

`KafkaConnector` resources provide the configuration needed to create connectors within a Kafka Connect cluster, which interacts with a Kafka cluster as specified in the `KafkaConnect` configuration. 
The Kafka cluster does not need to be managed by Strimzi or deployed to a Kubernetes cluster.

...

The configuration also specifies how the connector instances interact with external data systems, including any required authentication methods. Additionally, you must define the data to watch. 
For example, in a source connector that reads data from a database, the configuration might include the database name. 
You can also define where this data should be placed in Kafka by specifying the target topic name.